api/cloud/v1.md

```diff
@@ -74,9 +74,29 @@ docs_area: reference.api
   function changeVersion() {
     const doc = document.getElementById('rd');
     selectedVersion = document.getElementById("versionSelector").value;
-    doc.loadSpec('https://cockroachlabs.cloud/assets/docs/ccapi_spec_' + selectedVersion + '.json');
+    doc.loadSpec('https://cockroachlabs.cloud/assets/docs/' + selectedVersion + '.json');
   }
 
+  window.addEventListener('load', (event) => {
+    const params = new Proxy(new URLSearchParams(window.location.search), {
+      get: (searchParams, prop) => searchParams.get(prop),
+    });
+    // Get the value of the version query parameter
+    let version = params.version;
+    if (version != null) {
+      let select = document.getElementById("versionSelector");
+      // find the version by the option text (not the label)
+      for (var i = 0; i < select.length; i++){
+        var option = select.options[i];
+        if (version == option.text) {
```

Comment:
the 2 character comparison operators are generally not used anymore either.  Recommend replacing with === and !== (above).  These 3 character operators also check for type equality.  Do double check it still works as expected because the != null is probably a bit different than !== null.